devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -386,15 +487,15 @@ private def handleAccessAlarmReport(cmd) {
 			break
 		case 2: // Manually unlocked
 			map.descriptionText = "Unlocked manually"
-			map.data = [ method: "manual" ]
+			map.data = [ method: "manual", notify: true, notificationText: "$deviceName at ${location.name} was unlocked manually" ]
```

Comment:
What is the purpose of `notify` in the event data? Can't that be handled by the SmartApp?